portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -177,12 +177,48 @@ definition {
 	}
 
 	@description = "Verify that a Table can be created in the Body Text"
-	@ignore = "true"
 	@priority = "4"
 	test CanSupportTables {
 		property portal.acceptance = "false";
 
-		// TODO LPS-131236 CanSupportTables pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+		Click(
+			key_titleName = "Table",
+			locator1 = "CKEditor#TOOLBAR_ANY_ITEM_BUTTON");
+
+		Click(locator1 = "CKEditor#OK_BUTTON");
+
+		Form.save();
+
+		AssertElementPresent(locator1 = "Message#SUCCESS");
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		Click(
+			key_metadataSetName = "Form 1",
+			locator1 = "DocumentsAndMediaMetadataSets#METADATA_SETS_NAME");
+
+		AssertElementPresent(
+			key_fieldLabel = "Paragraph",
+			locator1 = "FormFields#LABEL");
+
+		AssertElementPresent(
+			key_fieldName = "liferay-ddm-form-field-paragraph-text",
+			locator1 = "FormFields#PARAGRAPH_BODY_CONTENT_TABLE");
```

Comment:
@piresdm, there is a macro which already does these 2 steps at once, please look for `FormFields.viewParagraphField`.

In addition, shouldn't we make sure that a `<table>...</table>` is also the content of the Body field? Otherwise, this assert is just checking that the Body field has some content. Does it make sense to you?

Reply:
@natocesarrego like we talked before, I am not using the macro because I am using a different path on the second assert to make sure there is a table inside.

Reply:
Thanks @natocesarrego and @piresdm . The path you're using identifies that there's a table on the builder.